_data/content/projects/fr.yml

```diff
@@ -5,228 +5,137 @@
+        l'industrie cinématographique espagnole, celles et ceux qui ne sont pas dans la lumière mais dans l'ombre, parmi eux, 
+        les cofondateurs de 8 y medio, María et son défunt mari, Jesús, leurs enfants et la mascotte du magasin, le petit chien Lucas, 
+        qui a guidé la visite de l'exposition dédiée aux enfants. Le message porté par l’exposition  est qu’aucune réussite ne pourrait 
+        exister sans les gens qui travaillent dur dans les coulisses.
+
+    - type: id_speak
+      project_name: "Nuevas expografías: entre la tecnología y las experiencias"
+      project_excerpt: "Séminaire du cycle de Colloques de Culture Visuelle Contemporaine"
+      project_ubication: "Fondation Mainel, Valence, Espagne"
+      img: ":22_Fundacion-Mainel_Coloquio-de-Cultura-Visual_Web.webp"
+      date: "Novembre 2022"
       post: |
-        Two things are infinite: the universe and human stupidity; and I'm not sure about the universe.
-    - type: id_quote
-      project_name: "Exemple de projet"
-      project_excerpt: "Mae West"
-      img: ":project1_thumb.jpg"
-      img_title: "img title5"
-      date: "2021-08-20"
+        Elisa a été invitée par la Fondation Mainel à participer à un panel de leaders du secteur de création de musées qui 
+        ont discuté du rôle des technologies et de la construction d’expériences dans la tenue d'expositions. Elle s'est concentrée 
+        sur diverses typologies d’expositions sur lesquelles elle a travaillé, depuis les expositions immersives à succès 
+        du V&A Museum de Londres (David Bowie Is, 'You Say You Want a Revolution ?' et Pink Floyd : Their Mortal Remains), 
+        les expositions internationales (en particulier celles de Dubaï Expo 2020) et ses expériences basées sur les 
+        technologies immersives. Le cycle de séminaires Culture Visuelle Contemporaine fait partie de l'offre publique de Mainel 
+        qui se concentre sur l'amélioration de la société à travers la solidarité et la culture comme outils de transformation, 
+        pour garantir que les personnes soient protagonistes de leur propre développement. Dans sa présentation, Elisa a montré 
+        comment les expériences immersives sont essentielles à la démocratisation de la culture, en transformant la visite passive 
+        en participation active, compréhension et une approche par les sentiments au même niveau que les connaissances partagées par 
+        les conservateurs et les institutions elles-mêmes. Les thèmes de cette discussion et les motivations derrière la programmation 
+        de Mainel montrent un profond souci du public, dont ils cherchent à enrichir et à responsabiliser la vie. 
+        Voir <a href="https://mainel.org/noticias/expografias-coloquio-cvc-27/" target="blank"> Nuevas expografías: se puede interactuar con la obra incluso sin que esté físicamente presente · Fundación Mainel </a>
```

Comment:
This link should use the same class as all the links in the page called: `link`, also do it at english and spanish version, this was a mistake.